README.md

```diff
@@ -11,7 +11,7 @@ Gidari is a "web-to-storage" tool for querying web APIs and persisting the resul
 
 ## Installation
 
-TODO
+go install github.com/alpine-hodler/gidari/cmd/gidari@latest
```

Comment:
This need to be formatted like this (with  "```"):

```
go install github.com/alpine-hodler/gidari/cmd/gidari@latest
```

We walso need to add this line under it:

Gidari is not currently available as a library. Library support will be investigated in https://github.com/alpine-hodler/gidari/issues/214#issue-1395594072